static/src/javascripts/projects/common/modules/commercial/build-page-targeting.ts

```diff
@@ -273,27 +271,14 @@ const buildAppNexusTargeting = once((pageTargeting: PageTargeting): string =>
 	formatAppNexusTargeting(buildAppNexusTargetingObject(pageTargeting)),
 );
 
-const getRdpValue = (ccpaState: boolean | null): string => {
-	if (ccpaState === null) {
-		return 'na';
-	}
-	return ccpaState ? 't' : 'f';
-};
-
 const consentedToAllPurposes = (consents: TCFv2ConsentList): boolean => {
 	return (
 		Object.keys(consents).length > 0 &&
 		Object.values(consents).every(Boolean)
 	);
 };
 
-const getTcfv2ConsentValue = (state: ConsentState | null): string => {
-	if (!state || !state.tcfv2) return 'na';
-
-	return consentedToAllPurposes(state.tcfv2.consents) ? 't' : 'f';
-};
-
-const getAdConsentFromState = (state: ConsentState | null): boolean => {
+const canTarget = (state?: ConsentState): boolean => {
```

Comment:
Might have misread this but is this function distinct from the way `canTarget` is computed when using enhanced consent? It feels like the logic is the same as that which is used to compute the `canTarget` parameters in `enhanceConsentState`?

Reply:
Ah yes good spot I was going to tackle in this PR.

`ConsentState` could definitely be replaced by `ConsentStateEnhanced` in which case `canTarget` is already available.

I'd now prefer a separate PR to do this as there a few other places `canTarget` is locally derived and I'd need to make sure that `ConsentStateEnhanced` was passed through to all those locations.

Reply:
That sounds good to me!